src/Components/Projects/index.js

```diff
@@ -15,6 +15,47 @@ function Projects() {
     >
       {/* <SectionHeader title="Partners" />
       <Sponsors /> */}
+      <View
+        style={{
+          marginTop: '1.2rem',
```

Comment:
we don't use `rem` as this will break the mobile native

Reply:
I come from react background that's why i have a tendency of using rem, i will change to % and aspect ratio